scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1335,7 +1357,19 @@ def query_object( self, kind, namespace, name ):
                              limit_once_per_x_secs=300, limit_key='k8s_api_build_query-%s' % kind )
```

Comment:
Same question here about `limit_key`

Reply:
Response same as above.

Reply:
This is slightly different due to the different logging level, so while I think the one above can have the limit removed, I think this limit is fine as is.  If it fails once here, it should fail in a predictable pattern so we don't need to see each individual instance of it.